### PR TITLE
docs: added drizzle doc page in english

### LINKED ISSUE #1613: feat: Update Drizzle document
### Is your feature request related to a problem? Please describe.

- Update the Drizzle documentation page
- Add Drizzle to why.md
- Add Drizzle to introduction.md

### Describe the solution you'd like to see

Create the Drizzle document based on the current Prisma document.

### Describe alternate solutions

N/A

### Additional information

I'll give contributing a try to the documents.  I'm new so please help me out if I am doing something wrong.  I read the contributions guideline and contributing guideline to docs.

### PR DESCRIPTION
**Pull request by ThomasBouasli**
Closes #1613

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_Added docs page for Drizzle ORM_

---

## Screenshots

_image_

💯

### CODE REVIEW

File: www/src/pages/en/usage/drizzle.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Drizzle
+description: Usage of Drizzle
+layout: ../../../layouts/docs.astro
+lang: en
+---
+
+Drizzle ORM is dialect specific, slim, performant and serverless ready by design.
```
Comment (barrybtw) [inline www/src/pages/en/usage/drizzle.md:8]: The wording here is a bit confusing, I suggest something like this?

Drizzle is a performant and slim ORM, that supports every major serverful and serverless runtime.

File: www/src/pages/en/usage/drizzle.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+You will find the Drizzle schema file at `src/server/db/schema.ts`. This file is where you define your database schema and models, but you can separate it into multiple files if you want, see how to do it at the [Drizzle schema documentation](https://orm.drizzle.team/docs/sql-schema-declaration). Drizzle is dialect specific and currently supports MySQL, PostgreSQL and SQLite.
+
+### With NextAuth.js
+
+When you select NextAuth.js in combination with Drizzle, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account`, and `VerificationToken` models, as per the [NextAuth.js documentation](https://authjs.dev/reference/adapter/drizzle).
+
+## Default Database
+
+The default database is an MySql database, using the PlanetScale serverless client, so it is edge ready out of the box! All you need to do is spin up a database at PlanetScale (up to one instance in free-tier BTW) and copy and paste the connection string in the `.env` file. Don't forget to add `ssl={"rejectUnauthorized":true}` to the end of the string.
```
Comment (barrybtw) [inline www/src/pages/en/usage/drizzle.md:24]: The default database is a* MySql database

**Review by barrybtw — CHANGES_REQUESTED** [review]
Small wording changes

File: www/src/pages/en/usage/drizzle.md (hunk 2 above)
Comment (ThomasBouasli) [reply www/src/pages/en/usage/drizzle.md:24]: Oops.

File: www/src/pages/en/usage/drizzle.md (hunk 1 above)
Comment (ThomasBouasli) [reply www/src/pages/en/usage/drizzle.md:8]: I took that description from Drizzle docs to be honest, I think it's fine but I don't mind changing it if you think its confusing.

Comment (barrybtw) [reply www/src/pages/en/usage/drizzle.md:8]: Oh, my mind inserted an 'a' after 'Drizzle ORM' haha, my mind was racing, didn't see what the noun was in the context before now 🥱😂

Comment (ThomasBouasli) [reply www/src/pages/en/usage/drizzle.md:8]: That's fine XD, I'll mark this as resolved than!

**Comment by ThomasBouasli** [comment]
Actually, don't merge this PR yet, just realize that I missed 2/3 topics in the original issue

- [x] Update the Drizzle documentation page
- [ ] Add Drizzle to why.md
- [ ] Add Drizzle to introduction.md

**Comment by ThomasBouasli** [comment]
I wont have the time to do this well enough, if anyone else wants to pick up where I left of feel free to do so.

**Pull request closed without merging**